packages/flutter_tools/test/general.shard/migrations/swift_package_manager_integration_migration_test.dart

```diff
@@ -671,6 +744,57 @@ void main() {
         expect(testLogger.traceText, contains('project.pbxproj already migrated. Skipping...'));
       });
 
+      testWithoutContext('skips PBXNativeTarget migration if all targets are migrated', () async {
```

Comment:
There was no "skip" test for when the Runner target was already migrated. This is one of two tests to cover this (still writing the other one at the moment)